### PR TITLE
QPD-1325: #comment Added non-mandatory fields logic to file_upload convert method

### PR DESCRIPTION
**Pull request by tushar-quartic**

### CODE REVIEW

**Review by lavkush-qu — APPROVED** [review]
lgtm

**Review by scriperdjq — APPROVED** [review]

File: graphene_django_extras/converter.py [hunk 1]
```diff
@@ -543,8 +543,9 @@ def convert_postgres_range_to_string(
...
 
 @convert_django_field.register(models.FileField)
-def convert_field_to_string(field, registry=None, input_flag=None, nested_field=False):
+def convert_field_to_string(field, registry=None, input_flag=None, nested_field=False,non_required_fields_list=[],field_name=None):
     return Upload(
         description=field.help_text or field.verbose_name,
-        required=is_required(field) and input_flag == "create",
+        required=is_required(
+            field) if not field_name in non_required_fields_list else False and input_flag == "create",
     )
```
Comment (shivinquartic) [inline graphene_django_extras/converter.py:551]: Please add an extra line here.

**Review by shivinquartic — APPROVED** [review]
lgtm

File: graphene_django_extras/converter.py (hunk 1 above)
Comment (tushar-quartic) [reply graphene_django_extras/converter.py:551]: Ok

**Pull request merged**